src/Resources/Resources/help/New-AzRoleAssignment.md

```diff
@@ -394,19 +394,17 @@ Accept wildcard characters: False
 ```
 
 ### -Scope
-The Scope of the role assignment.
-In the format of relative URI.
-For e.g.
-"/subscriptions/9004a9fd-d58e-48dc-aeb2-4a4aec58606f/resourceGroups/TestRG".
-If not specified, will create the role assignment at subscription level.
-If specified, it should start with "/subscriptions/{id}".
```

Comment:
Hey Stephan, as the description, if Scope is not specified, it defaults to the current subscription (you can get by `Get-AzContext`, so it is indeed an optional parameter.

Reply:
Hi @isra-fel , thank for responding. I tried assigning a new role to a service principal without specifying the -Scope option, but it did not work. The parameter is mandatory, hence my change in the documentation. 
![image](https://user-images.githubusercontent.com/57794508/141760327-d69beee9-eb5f-4ba0-9012-14f1efb23ec6.png)

Reply:
In this case you typed "-ObjectId" and "-RoleDefinitionId" so PowerShell knows you are calling this specific parameter set [RoleIdWithScopeAndObjectIdParameterSet](https://github.com/Azure/azure-powershell/blob/57b2691fccafc6796f35b77851f87e9a892a1058/src/Resources/Resources/help/New-AzRoleAssignment.md#roleidwithscopeandobjectidparameterset) , and `Scope` is required in it.
Some other parameter sets, like the [default one](https://github.com/Azure/azure-powershell/blob/57b2691fccafc6796f35b77851f87e9a892a1058/src/Resources/Resources/help/New-AzRoleAssignment.md#emptyparameterset-default), do not require `Scope` (notice the brackets around it), so overall it is optional. Hope I answered your question :)

Reply:
Okay. Thanks for the explanations :)